SimG4CMS/Calo/src/CaloSD.cc

```diff
@@ -107,6 +107,7 @@ CaloSD::~CaloSD() {}
 
 G4bool CaloSD::ProcessHits(G4Step* aStep, G4TouchableHistory*) {
   NaNTrap(aStep);
+  ignoreReject = false;
```

Comment:
Does this override the `ignoreRejection()` setter that is called elsewhere? If so, this line should probably be removed.

Reply:
This is the initialization of the flag ignoreReject. For certain cases it is called by some specific case - for HGCal it is done by setDetUnitId method.